TTBot/Commands/ChampionshipsModule.cs

```diff
@@ -179,41 +181,81 @@ public async Task GetStandings([Remainder] string args = null)
                     var results = await _results.GetChampionshipResultsByIdAsync(eventId);
                     var orderedResults = results.OrderBy(r => r.Pos);
 
-                    var resultsTable = orderedResults.ToStringTable(
-                        new[] { "Pos", "Driver", "Number", "Car", "Points", "Diff"},
-                            r => r.Pos, r => r.Driver, r => r.Number, r => r.Car, r => r.Points, r => r.Diff);
+                    string templateFilePath = @"assets/StandingsTemplate.png";
+                    Bitmap image = (Bitmap)Image.FromFile(templateFilePath);//load the image file
```

Comment:
Best having the image `using` statment rather than manually disposing incase there is an exception before it gets disposed.

Reply:
OK sweet I'll update it later